test/jdk/javax/net/ssl/SSLSession/ServerNameRejectedTLSSessionResumption.java

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
Please reformat to <= 80 chars.  This helps reviewers who prefer side/side reviews from having to horizontally scroll, even with large screens.  Thanks.